features/Generators.md

```diff
@@ -25,8 +25,5 @@ The same generator pattern is available for all specs:
 * feature
 * job
 * channel
-* generator
```

Comment:
I think that a generator generator is Rails-related, I remember a recent fix for its specs https://github.com/rspec/rspec-rails/pull/2217/files#diff-3fb1bfb25a4bda08b0ad2295bb53297aR3

Reply:
as I can see here: https://github.com/rspec/rspec-rails/tree/ca988b0697524717e41ce71663869708273b1966/lib/generators/rspec
Generators are aimed only for rails : )

So generate generator can be also included here.
I got a plugin by mistake from test unit 🤦 

I'm syncing all 15 generators with this list

Reply:
policy generator came from pundit gem

free to merge @pirj